src/app/resources/axis2/welcome.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html><head>
+<meta http-equiv="content-type" content="text/html; charset=windows-1252">
+    
+
+
+
+
+
+
+
+
+    <title>Login to Axis2 :: Administration page</title>
+    <link href="/resources/css/axis-style.css" rel="stylesheet" type="text/css">
+  </head>
+
+  <body onload="javascript:document.LoginForm.userName.focus();">
+    <!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements. See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership. The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License. You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<table summary="header table" class="FULL_BLANK">
+  <tbody><tr>
+    <td>
+	<table summary="header table" width="100%">
+	  <tbody><tr>
+	  	<td>
+		<table summary="embedded header table">
+  		  <tbody><tr>
+    		<td align="left"><img src="/resources/images/asf-logo.gif" alt="Apache Software Foundation Logo"></td>
+    	 	<td align="right"><img src="/resources/images/axis_l.jpg" alt="AXIS Logo"></td>
+          </tr>
+        </tbody></table>
+		</td>
+		<td>
+		&nbsp;
+		</td>
+	  </tr>
+	</tbody></table>
+
+    
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
why all these empty lines?

Reply:
The raw HTML file wasn't really good looking haha. Have fixed this.